.github/workflows/build-docker-image.yml

```diff
@@ -149,7 +159,7 @@ jobs:
         if: ${{ steps.image_build.outcome == 'success' }}
         uses: actions/upload-artifact@v3.1.0
         env:
-          VAPP_IMAGE: ${{ matrix.component.Name }}_${{ github.sha }}-${{ inputs.platform }}.tar
+          VAPP_IMAGE: ${{ matrix.component.Name }}-${{ inputs.platform }}
         with:
           name: ${{ env.VAPP_IMAGE }}
           path: ./${{ matrix.component.Name }}.tar
```

Comment:
Shouldn't this be the VAPP_IMAGE.tar?

Reply:
I thought the same but figured out that, this is a folder name that contains the {{appname}}.tar
- The structure will be like the seatadjuster-arm64/seatadjuster.tar